app/models/person.rb

```diff
@@ -50,7 +51,7 @@
 #  index_people_on_facebook_id_and_community_id  (facebook_id,community_id) UNIQUE
 #  index_people_on_id                            (id)
 #  index_people_on_reset_password_token          (reset_password_token) UNIQUE
-#  index_people_on_username_and_community_id     (username,community_id) UNIQUE
+#  index_people_on_username                      (username)
```

Comment:
Same here, the migrations don't touch to indexes, though there's a change here. But I guess this also fixes the state the index is.